### ISSUE TITLE
Remove `.py` from user-facing commands?

### ISSUE THREAD

**Issue by germasch** [wdmapp/effis#5]
When Sameer tried to build and run wdmapp today on Rhea, one of the many issues we hit was some confusion because some of the effis commands have the `.py` suffix, but others do not. (In particular, `effis-compose.py` vs `effis-submit`.) I can submit a PR to remove the `.py` from all, if @suchyta1 agrees?

**Comment by suchyta1** [comment]
Yes, I was planning to do that, or at least add links that don't have the `.py` extension. I can make that change if you don't beat me to it :)

**Comment by germasch** [comment]
Feel free to do it ;)

--Kai
